content/releases/Release Notes 09.25.25.md

```
@@ -0,0 +1,39 @@
+- Separated clean and claim work units in queue manager to prevent race conditions
+- Skip locked ActiveQueueSession rows on delete operations
+- Langfuse SDK integration updates for compatibility
+- Added configurable maximum message size to prevent token overflow in deriver
+- Various minor bugfixes
+
+## Links
+
+- [Sign-up for Honcho](https://app.honcho.dev/) & start building personalized agent experiences
+- [Join our Discord](https://discord.gg/honcho) & tell us what you're working on
+- [Visit our open-source repo](https://github.com/plastic-labs/honcho) & get your hands dirty
+- [Check out the docs](https://docs.honcho.dev)
```

markdownlint-cli2 (0.18.1)

Files should end with a single newline character

(MD047, single-trailing-newline)